packages/components/focus-trap/__tests__/focus-trap.spec.ts

```diff
@@ -105,7 +105,7 @@ describe('<ElFocusTrap', () => {
 
       const childComponent = findFocusComponent()
       const items = findDescendants()
-      expect(document.activeElement).toBe(items.at(0).element)
+      expect(document.activeElement).toBe(items.at(0)?.element)
```

Comment:
```suggestion
      expect(document.activeElement).toBe(items.at(0)!.element)
```

Reply:
I personally do not think that this is such a big deal in this case since if the assertion of `items.at(0)!.element` failed, it will be **Reference Error** instead of **Assertion Error**, when using `?` operator we can assure that this is not a syntax error.